honeypod/TcpdumpMonitor.py

```diff
@@ -0,0 +1,77 @@
+import subprocess
+import os
+
+
+class TcpdumpMonitor:
+    def __init__(self, api_key):
+        self.api_key = api_key
+
```

Comment:
Please pull in necessary code from main. Our interface and our IP address can change. Main already detects this:

#
# Get my interface; default eth0
#
network_interface = "eth0"
ip_address = ""


net_interfaces = psutil.net_if_addrs()

found = False
for interface, addresses in net_interfaces.items():
    for address in addresses:                                            
        if interface != "lo" and not address.address.startswith("127."): 
            network_interface = interface                                
            ip_address = address.address
            found = True
            break
    if found:
        break